documentation/guidelines.md

```diff
@@ -0,0 +1,303 @@
+
+### Suggestions and comments
+
+Some notes about the relationship between suggestions and comments:
+
+Annotation bodies may contain both suggestions and comments. By including both, an annotation can suggest new data while also including a comment explaining the reason for the correction.
+
+The relationship between suggestions/comments in the body and the motivation is somewhat loose. It does not make much sense to make `oa:motivatedBy oa:editing` annotation without a suggestion.
+
+However, it is conceivable to make a `oa:motivatedBy oa:commenting` annotation that also includes suggestions. In that case, the suggestions may be thought of as structured references to predicates, rather than propositions that those predicates hold for the subject. This could be used to express uncertainty, e.g. for annotations like "I think it's something like this". However, an application may ignore this possibility.
+
+### Predicate selectors and predicate suggestions
+
+We have seen that annotation targets can have a predicate selector and annotation bodies can make suggestions about predicates. It is therefore possible to make an annotation that has both, e.g.:
+
+```ttl
+[]  a edpopcol:Annotation ;
+    oa:motivatedBy oa:editing ;
+    oa:hasTarget [
+        oa:hasSource _:Denmark ;
+        oa:hasSelector [
+            edpopcol:property _:king ;
+            edpopcol:object _:HamletSenior
+        ]
+    ] ;
+    oa:hasBody [
+        edpopcol:suggestsRemoval [
+            edpopcol:property _:king ;
+            edpopcol:object _:HamletSenior
+        ] ;
+        edpopcol:suggestsAddition [
+            edpopcol:property _:king ;
+            edpopcol:object _:Claudius
+        ]
+    ] .
+```
+
+One could argue that the predicate selector is redundant: we can already read from the suggestion that this annotation is correcting the predicate `_:king _:HamletSenior`.
+
+Nonetheless, it can be useful to include target selectors for suggestions. One reason is that the redundant information can make querying a bit easier. 
+
+Note that the selector describes the current state of the graph: the annotation is reacting to the triple `_:Denmark _:king _:HamletSenior`. It should not select the suggested predicate `_:king _:Claudius`.
+
+## Suggestions and changing the graph
+
+Annotations can provide structured suggestions; whether these suggestions are accepted is another question.
+
+#### Accepting corrections
+
+It is useful to express that an annotation is regarded as canonical in a particular context. For instance, when a researcher makes corrections within their own collections, those may be regarded as canonical in that context. A reasoner could then use this status to create a unified, corrected graph.
+
+At the moment, there is no implementation for this within the vocabulary, due its complexity. Canonical status raises questions about moderation, and creating a unified graph is a non-trivial task, since annotations can contradict each other. 
```

Comment:
```suggestion
At the moment, there is no implementation for this within the vocabulary, due to its complexity. Canonical status raises questions about moderation, and creating a unified graph is a non-trivial task, since annotations can contradict each other. 
```